components/arm/arm.go

```diff
@@ -187,16 +188,19 @@ func CheckDesiredJointPositions(ctx context.Context, a Arm, desiredInputs []refe
 		max := limits[i].Max
```

Comment:
I also could see an argument for changing the function signature to accept *pb.JointPositions instead of []referenceframe.Input since that way we are returning the error in the units in the same units they are provided.  But I'm not sure this is a great idea since it seems likely we will continue to prefer Inputs over pb.JointPositions as we continue to develop arms and would rather avoid the extra breaking change to the package and modules that depend on it.

Reply:
I think the cost of having a slightly unintuitive error here is probably worth keeping the package contract stable

Reply:
my opinion is that we should not accept `*pb.JointPositions` as an argument into functions, so we should keep it as is.